include/IGPUMappedBuffer.h

```diff
@@ -17,10 +17,13 @@ namespace video
 class IGPUMappedBuffer : public virtual video::IGPUBuffer
 {
     public:
-        //! WARNING: RESIZE will invalidate pointer
-        //! WARNING: NEED TO FENCE BEFORE USE!!!!!!!!!!!!!
+		//! Gets internal pointer.
+        /** WARNING: RESIZE will invalidate pointer!
+        WARNING: NEED TO FENCE BEFORE USE!
+		@returns Internal pointer. */
```

Comment:
Its best you use a GPU Fence to ensure any operations which you've queued up that are writing to this buffer or reading from it have completed before you start using this pointer, otherwise you will have a race condition.